web_src/js/index.js

```diff
@@ -996,17 +996,21 @@ async function initRepository() {
               $renderContent.html(data.content);
             }
             const $content = $segment.parent();
-            if (!$content.find('.ui.small.images').length) {
+            if (!$content.find('.dropzone-attachments').length) {
               if (data.attachments !== '') {
                 $content.append(
-                  '<div class="ui bottom attached segment"><div class="ui small images"></div></div>'
+                  `<div class="dropzone-attachments">
+                  <div class="ui clearing divider"></div>
+                  <div class="ui middle aligned padded grid">
+                  </div>
+                </div>`
```

Comment:
Intend like this:

```js
$content.append(`
  <div class="dropzone-attachments">
    <div class="ui clearing divider"></div>
    <div class="ui middle aligned padded grid"></div>
  </div>
`);

Reply:
That's just going to insert lots of empty new lines.

What's the benefit?